conanfile.py

```diff
@@ -26,14 +27,17 @@ def configure(self):
             self.output.warn("Chromium depot_tools is not well supported by Python 3!")
 
     def source(self):
-        commit = "e5641be5fe309f40aad850d4d1e1ca607768572c"
-        tools.mkdir(self._source_subfolder)
-        with tools.chdir(self._source_subfolder):
-            tools.get("{}/+archive/{}.tar.gz".format(self.homepage, commit))
+        commit = "cc6f585f055ae696170b22f0e8db906d27afe636"
+        git = Git(folder=self._source_subfolder)
```

Comment:
Why this change to a git checkout?

Reply:
not sure, but it doesn't make sense. I gonna revert to old style. Thanks for reviewing it.